### DISCUSSION TITLE
Add Translation String on to Flow Description

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#13911] [Feature Requests]
While going through the app to start the translation Strings draft, it feels like we may want to add a translation string option onto Flows. Just a thought.

<img width="1584" alt="Screen Shot 2022-06-15 at 1 28 54 PM" src="https://user-images.githubusercontent.com/62306630/173899097-ce1d47d0-a852-45b3-bde1-c79e218be45f.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
